### PR TITLE
Colin changes new office

### PR DESCRIPTION
**Pull request by peter-freshworks**
*Issue #:* /bcgov/entity1597

*Description of changes:*
change calls to Oracle to include the office type 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: coops-ui/tests/unit/Dashboard.spec.ts [hunk 1]
```diff
@@ -114,11 +114,11 @@ describe('Dashboard - UI', () => {
     // emit filing list from Filing History component with the completed filing
     wrapper.find(FilingHistoryList).vm.$emit('filings-list', [
       {
-        'name': 'Director Change',
```
Comment (thorwolpert) [inline coops-ui/tests/unit/Dashboard.spec.ts]: This file shouldn't be in your PR.
Please remove , and remember to not change others spacings.

File: entity-filer/src/entity_filer/service.py [hunk 2]
```diff
@@ -32,6 +32,13 @@
 class ServiceWorker():
```
Comment (thorwolpert) [inline entity-filer/src/entity_filer/service.py]: Remove this file from the PR, it does not pertain to the COLIN work.

File: schemas/tests/unit/test_filings.py [hunk 3]
```diff
@@ -103,6 +104,68 @@ def test_invalid_coa_filing_bcorp():
...
+        'filing': {
+            'header': {
+                'name': 'changeOfDirectors',
+                'date': '2019-04-08',
+                'certifiedBy': 'full legal name',
+                'email': 'no_one@never.get'
+            },
+            'business': {
+                'cacheId': 1,
+                'foundingDate': '2007-04-08',
```
Comment (thorwolpert) [inline schemas/tests/unit/test_filings.py]: Add timestamp to foundingDate
'foundingDate': '2007-04-08T00:00:00+00:00'

File: schemas/tests/unit/test_filings.py [hunk 4]
```diff
@@ -103,6 +104,68 @@ def test_invalid_coa_filing_bcorp():
...
```
Comment (thorwolpert) [inline schemas/tests/unit/test_filings.py]: Add timestamp to foundingDate
'foundingDate': '2007-04-08T00:00:00+00:00'

File: coops-ui/tests/unit/Dashboard.spec.ts (hunk 1 above)
Comment (peter-freshworks) [reply coops-ui/tests/unit/Dashboard.spec.ts]: weird...I didn't touch this

**Pull request merged**